enclave-manager/server/server.go

```diff
@@ -298,6 +298,31 @@ func (c *WebServer) GetEnclaves(ctx context.Context, req *connect.Request[emptyp
 	}
 	return resp, nil
 }
+
+func (c *WebServer) GetEnclavesByUuids(ctx context.Context, req *connect.Request[kurtosis_enclave_manager_api_bindings.GetEnclavesByUuidsRequest]) (*connect.Response[kurtosis_engine_rpc_api_bindings.GetEnclavesResponse], error) {
```

Comment:
a bit confused: I thought the result of our last chat was that you wanted to add a filters parameter to GetEnclaves, rather than adding a new endpoint? but I trust you know what you're doing